cmd/glasskube/cmd/version.go

```diff
@@ -33,3 +33,16 @@ var versioncmd = &cobra.Command{
 func init() {
 	RootCmd.AddCommand(versioncmd)
 }
+
+func createAsciiText() string {
+	blue := "\033[34m"
+	white := "\033[0m"
+
+	return fmt.Sprintf(`%s
+  ____ _        _    ____ ____  _  ___   _ ____  _____ 
+ / ___| |      / \  / ___/ ___|| |/ / | | | __ )| ____|
+| |  _| |     / _ \ \___ \___ \| ' /| | | |  _ \|  _|  
+| |_| | |___ / ___ \ ___) |__) | . \| |_| | |_) | |___ 
+ \____|_____/_/   \_\____/____/|_|\_\\___/|____/|_____|
```

Comment:
This should be a constant. 

Also, we already use the `color` package, so manually adding control sequences should be avoided. Something like `color.Blue(asciiLogo)` should work (where `asciiLogo` is the name of the string constant